azext_iot/operations/dps.py

```diff
@@ -18,36 +18,41 @@
 from azext_iot.common.utility import compute_device_key
 from azext_iot.operations.generic import _execute_query
 from azext_iot._factory import SdkResolver
-from azext_iot.sdk.dps.models.individual_enrollment import IndividualEnrollment
-from azext_iot.sdk.dps.models.custom_allocation_definition import (
+from azext_iot.sdk.dps.service.models.individual_enrollment import IndividualEnrollment
```

Comment:
Because the required models here are being imported on models `__init__`, you should be able to import straight from .`models` as in `from azext_iot.sdk.dps.service.models import IndividualEnrollment` and so on